### PR TITLE
2 service deploy fixes

### PR DESCRIPTION
**Pull request by coilysiren**
## Context

2 quick infra fixes to unblock prod deploy

## Testing

I deployed this to dev already to test

### CODE REVIEW

**Review by lucasmbrown-usds — APPROVED** [review]
I used to be a pretty good SRE - now my opinion should be taken with a grain of salt!

File: infra/modules/service/main.tf [hunk 1]
```diff
@@ -9,9 +9,9 @@ data "external" "whoami" {
   program = ["sh", "-c", "whoami | xargs -I {} echo '{\"value\": \"{}\"}'"]
 }
 
-data "external" "deploy_github_ref" {
-  program = ["sh", "-c", "git branch --show-current | xargs -I {} echo '{\"value\": \"{}\"}'"]
-}
+# data "external" "deploy_github_ref" {
```
Comment (lucasmbrown-usds) [inline infra/modules/service/main.tf:12]: Maybe add a note with a ticket number or another comment of TODO, explaining how and when we'll fix this?

Comment (coilysiren) [reply infra/modules/service/main.tf:12]: That'll reset the review so I'll fast follow

Comment (coilysiren) [reply infra/modules/service/main.tf:12]: https://github.com/HHS/simpler-grants-gov/issues/3177

**Review by mdragon — APPROVED** [review]

File: infra/modules/service/main.tf (hunk 1 above)
Comment (lucasmbrown-usds) [reply infra/modules/service/main.tf:12]: Perfect, thanks!!

**Pull request merged**